consoles/sshVirtsh.pm

```diff
@@ -112,6 +112,22 @@ sub _init_xml ($self, $args = {}) {
     $elem = $doc->createElement('pae');
     $features->appendChild($elem);
 
+    ## Following features are required for s390's SIE instruction
+    # https://share.confex.com/share/117/webprogram/Handout/Session9564/9564%20-%20zVM%20Security%20and%20Integrity.pdf p6
+    # https://qemu.readthedocs.io/en/latest/system/s390x/vfio-ap.html#start-interpretive-execution-sie-instruction
+    
+    if ( defined $bmwqemu::vars{S390_SIE_AP_MATRIX_PATH} ){
+        $elem = $doc->createElement('ap');
+        $features->appendChild($elem);
```

Comment:
I know you're just copying from above, but I'd prefer:
```suggestion
        $features->appendChild($doc->createElement('ap'));
```

Reply:
Agreed. Reusing the same variable over and over is not nice.